packages/procaptcha-react/src/components/CaptchaComponent.tsx

```diff
@@ -28,135 +28,143 @@ export interface CaptchaComponentProps {
+                        <Typography
+                            px={1}
                             sx={{
-                                display: 'flex',
-                                alignItems: 'center',
-                                justifyContent: 'center',
-                                width: '100%',
+                                color: '#ffffff',
+                                fontWeight: 700,
+                                textTransform: 'uppercase',
+                                fontSize: theme.typography.h6.fontSize,
                             }}
-                            {...addDataAttr({ dev: { cy: 'dots-captcha' } })}
                         >
-                            {challenge.captchas.map((_, i) => (
-                                <Box
-                                    key={i}
-                                    sx={{
-                                        width: 7,
-                                        height: 7,
-                                        borderRadius: '50%',
-                                        border: '1px solid #CFCFCF',
-                                    }}
-                                    mx={0.5}
-                                    bgcolor={index === i ? theme.palette.background.default : '#CFCFCF'}
-                                />
-                            ))}
-                        </Box>
-                        <Box
-                            px={2}
-                            pt={0}
-                            pb={2}
-                            sx={{
-                                display: 'flex',
-                                alignItems: 'center',
-                                justifyContent: 'space-between',
-                            }}
+                            {`${at(challenge.captchas, index).captcha.target}`}
+                        </Typography>
+                    </Box>
+
+                    <Box {...addDataAttr({ dev: { cy: 'captcha-' + index } })}>
+                        <CaptchaWidget challenge={captcha} solution={solution} onClick={onClick} />
+                    </Box>
+                    <Box
+                        px={2}
+                        py={1}
+                        sx={{
+                            display: 'flex',
+                            alignItems: 'center',
+                            justifyContent: 'center',
+                            width: '100%',
+                        }}
+                        {...addDataAttr({ dev: { cy: 'dots-captcha' } })}
+                    >
+                        {challenge.captchas.map((_, i) => (
+                            <Box
+                                key={i}
+                                sx={{
+                                    width: 7,
+                                    height: 7,
+                                    borderRadius: '50%',
+                                    border: '1px solid #CFCFCF',
```

Comment:
should the colour come from the theme?

Reply:
Could do - What colour do you want to go for the image border for dark mode?

Personally I'm not completely convinced that the images really need borders, so happy with whatever you want to go for on this one